pkgs/development/tools/minizinc/default.nix

```diff
@@ -1,17 +1,31 @@
-{ lib, stdenv, fetchFromGitHub, cmake, flex, bison }:
-stdenv.mkDerivation rec {
+{ lib, stdenv, fetchFromGitHub, callPackage, jq, cmake, flex, bison, gecode, mpfr, cbc, zlib }:
+stdenv.mkDerivation (finalAttrs: {
   pname = "minizinc";
   version = "2.7.4";
 
-  nativeBuildInputs = [ cmake flex bison ];
+  nativeBuildInputs = [ cmake flex bison gecode mpfr cbc zlib ];
 
   src = fetchFromGitHub {
     owner = "MiniZinc";
     repo = "libminizinc";
-    rev = version;
+    rev = finalAttrs.version;
     sha256 = "sha256-Zq5gAwe9IQmknSDilFyHhSk5ZCQ8EfBOiM6Oef2WxYg=";
   };
 
+  postInstall = ''
+    mkdir -p $out/share/minizinc/solvers/
+    ${jq}/bin/jq \
```

Comment:
This should be added to nativeBuildInputs instead.

Reply:
As in, make a derivation that contains just the config file?  Do you have an example that does this, by any chance?

Reply:
I fixed my comments in https://github.com/NixOS/nixpkgs/pull/235439

Reply:
Got it.

I see you moved some of the definitions around.  Was this a tool reformatting the file?

Reply:
Done by hand.